packages/playwright-test/src/testLoader.ts

```diff
@@ -34,7 +34,7 @@ export class TestLoader {
     this._fullConfig = fullConfig;
   }
 
-  async loadTestFile(file: string, environment: 'runner' | 'worker'): Promise<Suite> {
+  async loadTestFile(file: string, environment: 'loader' | 'worker'): Promise<Suite> {
```

Comment:
Unrelated, but perhaps we should take `loadErrors` array here as in other places, instead of setting `suite._loadError` flag?

Reply:
yeah, i can follow up